includes/functions-upgrade.php

```diff
@@ -27,6 +30,9 @@ function yourls_upgrade( $step, $oldver, $newver, $oldsql, $newsql ) {
 		if( $oldsql < 482 )
 			yourls_upgrade_482();
 
+		if( $oldsql < 505 )
+			yourls_upgrade_to_18();
```

Comment:
```suggestion
			yourls_upgrade_to_505();
```

For consistency 🙂

Reply:
Actually I think this is `yourls_upgrade_482()` that was inconsistent. Much easier to see what has changed from the version number/tag than to git blame and check when that arbitrary number was bumped